lldb/source/Target/Target.cpp

```diff
@@ -782,7 +824,7 @@ void Target::ApplyNameToBreakpoints(BreakpointName &bp_name) {
 
 void Target::GetBreakpointNames(std::vector<std::string> &names) {
   names.clear();
-  for (const auto& bp_name_entry : m_breakpoint_names) {
+  for (const auto &bp_name_entry : m_breakpoint_names) {
```

Comment:
All the formatting below seems unrelated to your PR, probably because you ran `clang-format` on the whole file. We try to avoid this because is overwrites the file history with the actual explanations of what previous contributors changes, with your commit which is not related.

If you want to reformat the rest of the file, that should be done in a separate commit and you should add the commit hash to the [.git-blame-ignore-revs](https://github.com/llvm/llvm-project/blob/main/.git-blame-ignore-revs) file.

Now, if you want to address this, you can do the following:

0. Save your current branch in case things go wrong with git:
```
$ git checkout -b stop-at-user-entry-formatted
```

1. Un-do commit without deleting your work.
```
$ git reset HEAD~1
```
2. Add only changes related to the PR (skip all the formatting changes).
```
$ git add -p .
```
3. Make sure the changes that are staged don't contain reformatting changes.
```
$ git status
$ git commit -v
```
4. Discard the formatting changes.
```
$ git reset --hard HEAD
```

If you want to format specifically what you changed in your commit, you should run:
```
$ git clang-format HEAD~
$ git add -u
$ git commit --amend --no-edit
```
Make sure you don't have any other unstaged changes before doing this, otherwise, they'll be merged with the formatting changes.

Let me know if you need help or have any questions.

Reply:
I was wondering if that was a problem, thank you for clarifying and for the step-by-step!

Should I squash the two commits again?

Reply:
> Should I squash the two commits again?

You can if you want.